### DISCUSSION TITLE
GitHub notification is bugged in iOS

### DISCUSSION THREAD

**Discussion by glekner** [community/community#77085] [Mobile]
![image](https://github.com/community/community/assets/24938324/dfb562c3-504a-4a3a-8cfb-fe5f2f455dbc)
<details>
<summary>ℹ️ Additional Debug Info</summary>
GitHub for iOS, Version 1.139.1 (116337430)
Device: iPhone16,1 (iOS 17.1.1)
</details>

**Comment by HenSquared** [comment]
hi @glekner, thanks for sharing the screenshot

Can you share additional screenshots of how the notification looks on your device? 
If possible, please also include any steps to help us reproduce the issue. 

Thanks for the feedback!
